draco/synthesis/mockcatalog.py

```diff
@@ -374,12 +391,17 @@ def make_pdf_map(self, source_map, z_weights, selfunc=None):
             nside=source_map.nside,
             polarisation=False,
             freq=source_map.index_map["freq"],
+            attrs_from=selfunc,
         )
 
         # Put computed PDF into local section of container
         pdf_map_local = pdf_map.map[:]
         pdf_map_local[:, 0, :] = pdf
 
+        # Set a tracer attribute
+        if self.tracer is not None:
+            pdf_map.attrs["tracer"] = self.tracer
```

Comment:
This will override the `tracer` attribute inherited from the selection function (if any), but I guess that's fine

Reply:
Yeah. I think that's generally the expected behaviour everywhere, i.e. if you explicitly set anything then it's expected to take precedence and overwrite anything else.